docs/src/02-learn/migration/index.md

```diff
@@ -1,30 +1,51 @@
 # Migration
 
-A big effort is made to assure that a new CLI release does not break previous projects scaffolding and maintain compatibility with previous MES releases.
+A big effort is made to ensure that a new CLI release does not break previous projects' scaffolding and maintains compatibility with previous MES releases.
 
-Nevertheless, please consider the instructions in this guide when upgrading the CLI for your project.
+Nevertheless, please consider the conventions and instructions in this guide when upgrading the CLI for your project.
 
-## Minor versions upgrade
+## Conventions
 
-Minor versions upgrade should not affect your project, only provides enhancements or fixes that should be retro-compatible.
+### Minor versions upgrade
 
-If you found an issue, open a Support Request on CM Customer Portal or send us a pull request with a fix proposal.
+Minor CLI version upgrades should not affect your project, only provide enhancements or fixes that should be retro-compatible.
 
-## Major versions upgrade
+If you found an issue, open a Support Request on the CM Customer Portal or send us a pull request with a fix proposal.
 
-The CLI major version is incremented every time there are breaking changes on the CLI. On such case, if you decide to upgrade to a new release, please execute the following:
+### Major versions upgrade
+
+The CLI major version is incremented every time there are breaking changes on the CLI. In such case, if you decide to upgrade to a new release, please execute the following:
 
 * Review the release log to get hints on what was changed;
+* Install the new CLI;
 * Re-scaffold your project;
 * Merge your customization changes into the new project structure.
 
-Please, review some version specific migration details:
+Please, review some version-specific migration details:
 
 * [Migration to V3 or above](migration-v3.md)
 
-## Release Tags
+### Release Tags
 
 The @criticalmanufactuing/cli has release tags that you may use to dynamically retrieve the latest @releases:
 
 * __latest__: tag applied to the latest stable package that has been released;
-* __vnext__: tag applied to the latest release, usually used by the CLI early adopters to use and test the latest features/fixes.
+* __next__: tag applied to the latest release, usually used by the CLI early adopters to use and test the latest features/fixes.
+
+## Upgrade Step-by-Step
+
+To upgrade the @criticalmanufacturing/cli:
+
+* Open a command line with administration privileges;
```

Comment:
Admin rights are not required.

Reply:
We recommend installing globally for easier access. If you don't have admin rights, you'll may need to adjust the installation path based on Node setup. As so, I would keep current doc as is.